proposals/PR21 - Proposal Documentation and Review.md

```diff
@@ -0,0 +1,192 @@
+   - For example: if your proposal is titled `Process for XYZ` and the prior approved SIP number was `012`, then your new file name would be `013 - Process for XYZ.md`.
+3. Add the new document title, with hyperlink, to the "Approved" section of the index (`proposals/index.md`).
+4. Merge the pull request.
+
+If the SIP is not approved, it will return to draft status and/or it may be replaced by a new or altered submission based on Working Group feedback.
+
+### Document Lifecycle Edits
+
+Occasionally, existing Proposal documents may need to be edited or updated for readability or clarification. Modifications to approved proposals will be managed using the following process:
+
+1. Any Member Company may propose edits or clarifications by submitting a pull request against the document.
+2. The author shall add a "Change log" section to the document, with an entry describing the nature of the update(s).
+3. The author should request approval from one or more Leading Members.
+4. The pull request may be merged by the author, pending at least one approval on the pull request from a Leading Member.
+
+This Edit process _should not_ be used for any significant changes which would significantly alter or undo prior decisions. Significant modifications and revisions shall require a fresh vote, and any Leading Member may reject the proposed edit and/or request the edit be submitted as a formal Proposal.
+
+Examples of valid lifecycle edits:
+
+1. Expanding upon documentation which may have otherwise been unclear.
+2. Correcting small grammatical or spelling issues.
+3. Creating or updating section headers for readability.
+4. Improving readability by providing examples of a specific use case or behavior.
+5. Adding hyperlinks and references to related resources.
+6. Adding or updating a table of contents.
+7. Minor updates to working group processes, especially if those updates don't impact general function or outcomes. (For example, extending a review period from 14 days to 21 days.)
+
+Examples of significant edits which should instead be submitted as formal proposals:
+
+1. Updating from one standard framework to a newer version of the same standard.
+1. Introducing significant changes to key processes or to a critical function of the spec.
+1. Any other substantive change that could impact the stability or interoperability of the spec.
+
+-----------------------
+
+## III. Additional Information
+
+<!-- Note: Author may delete any headers in this section which are not relevant. -->
+
+### Which layer(s) of the Singer ecosystem does this proposal directly touch?
+
+Select all that apply:
+
+- [ ] Singer Specification - required capabilities and behaviors
+- [ ] Singer Specification - optional capabilities and behaviors
+- [ ] Singer best practices and other guidance
+- [x] **Singer Working Group - practices and procedures**
+- [ ] Singer documentation (Other)
+
+### What are the downsides to this change, if any?
```

Comment:
As mentioned in the Feb meeting, I'd like to propose that this not have any sort of language like "if any" to strongly encourage documenting the thought process that went through thinking about downsides, since there are always some kinds of downsides to any change (e.g., influencing a process towards a specific direction, less flexibility in some use cases, effects if someone is using a different language/platform/orchestration tech, etc.), and perhaps connecting this to alternatives, potential weaknesses, etc.

In this proposal, there aren't many clear downsides, other than the ever present scary unknowns. It might be worth poking holes in the potential weaknesses of the voting approach, maybe what happens if this process turns out to be too rigid, any potential for locking proposals in a back and forth, etc.

Otherwise, at least acknowledging the research that went into this SIP and the feeling that, given that, it is at least a good starting point would be good to mention here I think :)